packages/notification-center/src/i18n/lang.ts

```diff
@@ -25,6 +25,7 @@ import { CS } from './languages/cs';
 import { PU } from './languages/pu';
 import { TA } from './languages/ta';
 import { KO } from './languages/ko';
+import { KU } from './languages/ki';
```

Comment:
```suggestion
import { KU } from './languages/ku';
```
File name should be `ku.ts`.